oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/util.js

```diff
@@ -200,3 +201,29 @@ export const getValidTagsForEditor = (tags, attr) => {
 
   return result.join(",");
 };
+
+export const serializeErrors = (
+  errors,
+  message = i18next.t(
+    "Draft saved with validation errors. Fields listed below that failed validation were not saved to the server"
+  )
+) => {
+  if (errors?.length > 0) {
+    let errorsObj = {};
+    const errorPaths = [];
+    for (const error of errors) {
+      errorsObj = setIn(errorsObj, error.field, error.messages.join(" "));
```

Comment:
Since this comes from user-input url #aaaaa  string, are messages sanitized before displaying in form (no-html/script should be allowed there)?

Reply:
This function just serves to package the errors received from the server into a shape that is useful for formik's errors and for the FormFeedback cmp. The errors are received from the server and they just contain the field path for the field that resulted in error and the error message (which is array of strings) (provided by BE). No user input exists there. If I misunderstood the question, we can do a call